src/lit-element.ts

```diff
@@ -41,6 +41,14 @@ export interface CSSResultArray extends Array<CSSResult|CSSResultArray> {}
  */
 const renderNotImplemented = {};
 
+/**
+ * Base element class that manages element properties and attributes, and
+ * renders a lit-html template.
+ * 
+ * To define a component, subclass `LitElement` and implement a
+ * `render` method to provide the component's template. Define properties 
+ * using the [[`properties`]] property or the [[`property`]] decorator. 
```

Comment:
Do we need both the `[[` and the ` here?

Reply:
If we use just `[[`, we get the link in regular text font. To my eye it looks a little funny if we have the API members in code font everywhere _except_ in links.